joomla-projects/gsoc17_helpscreens_on_jdocs
# Integration of Button with the Guided Tour Using JSON

Pull Request for Issue #6  .

### Summary of Changes
Currently, we have only the Help Screens to help new Joomla! Users to get an overview of any component’s view. To help make it easier and more interactive for new users, this PR has been submitted.

### Testing Instructions
1)Apply the Patch.

2)Enable the plugin 'plg_system_tour'
   If you seem not able to get Plugin do download from here ( https://drive.google.com/open?id=0B1eYyialFUnoZ0EzNWdWVjc3cUk ).
OR
We can Navigate to Extension → Manage → Discover → 
![plgin dicover](https://user-images.githubusercontent.com/17943374/29527562-e386e872-86b6-11e7-9f67-c4a07effec1b.png)

![discover plugin](https://user-images.githubusercontent.com/17943374/29527589-ff010420-86b6-11e7-8e44-07adcab95faf.png)
Select the Plugin “plg_system_tour” and Install it.
![plugin screenshot enable](https://user-images.githubusercontent.com/17943374/29267320-cc375a3c-8105-11e7-8f54-0533462c95eb.png)

3)Once the Plugin is enabled you can see the Button name "Take the Tour" 
![joomla2](https://user-images.githubusercontent.com/17943374/29267376-0dd2826e-8106-11e7-9ad0-4f32b06bc117.png)  at the Top right corner of the View.

4)If the file of that view exist tour will be started, else the Guided tour will point
 on Help screen Button 
![file not exist guided tour](https://user-images.githubusercontent.com/17943374/29267473-8187e4b0-8106-11e7-8a27-876eb011ec2e.png)
 .
Video for the Overview of the Guided Tour (
https://youtu.be/XWLC19FSwTM )


### Expected Outcome
The "Take the Tour" Button will exist after applying the Plugin and if the JSON file for that View exist the Tour will be started else it will point towards the Help Screen Button.
